### PR TITLE
Feature/eskl 170 editor testing

### PR DESCRIPTION
**Pull request by antoniomsgneto**

### CODE REVIEW

File: Assets/Scripts/Eskills/EskillsService.cs [hunk 1]
```diff
@@ -55,6 +67,19 @@ public void GetRoomInfo(string session, Action<RoomData> success, Action<Eskills
             _eskillsManager.SetScore(session, status, score, success, error);
         }
 
+        public void GetTicket(string ewt, string ticket_id, Action<TicketData> success, Action<EskillsError> error)
+        {
+            _eskillsManager.GetTicket(ewt, ticket_id, success, error);
+        }
+
+        public void JoinQueue(string ewt, MatchParameters matchParameters)
```
Comment (Trinkes) [inline Assets/Scripts/Eskills/EskillsService.cs]: should this method be public?

File: Assets/Scripts/Eskills/EskillsService.cs [hunk 2]
```diff
@@ -55,6 +67,19 @@ public void GetRoomInfo(string session, Action<RoomData> success, Action<Eskills
...
+        {
+            _eskillsManager.JoinQueue(ewt, matchParameters, session => {
+                Debug.Log(session);
+                GetComponent<OnMatchCreatedReceiver>().OnMatchCreated(session);
+                }, error => {});
```
Comment (Trinkes) [inline Assets/Scripts/Eskills/EskillsService.cs]: you might want to log the error or give the error feedback back to the developer somehow

File: Packages/packages-lock.json [hunk 3]
```diff
@@ -1,13 +1,15 @@
 {
   "dependencies": {
     "com.unity.2d.animation": {
-      "version": "3.2.2",
+      "version": "5.0.9",
```
Comment (Trinkes) [inline Packages/packages-lock.json]: Do you really want to increase unity version?

**Review by Trinkes — APPROVED** [review]

File: Assets/Scripts/Eskills/EskillsService.cs (hunk 1 above)
Comment (antoniomsgneto) [reply Assets/Scripts/Eskills/EskillsService.cs]: You are right, changed to private

File: Assets/Scripts/Eskills/EskillsService.cs (hunk 2 above)
Comment (antoniomsgneto) [reply Assets/Scripts/Eskills/EskillsService.cs]: logging the error

File: Packages/packages-lock.json (hunk 3 above)
Comment (antoniomsgneto) [reply Packages/packages-lock.json]: Changed back to the previous version

**Pull request closed without merging**